public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanDetail/index.tsx

```diff
@@ -326,8 +341,11 @@ export default function SpanDetail(props: SpanDetailProps) {
           <LabeledList className={styles.list} divider={true} items={overviewItems} />
         </div>
       </div>
-      <span style={{ marginRight: '10px' }}>{logLinkButton}</span>
-      {profileLinkButton}
+      <div className={styles.linkList}>
+        {logLinkButton}
+        {profileLinkButton}
+        {sessionLinkButton}
+      </div>
```

Comment:
Humm I know what you mean, I added the flex wrap so that in worst case we're at least not hiding any button

Reply:
Plugin extension for trace links may be worth exploring

Reply:
I'm also curious why we're not going with a plugin [extension](https://grafana.com/developers/plugin-tools/ui-extensions), mostly the new way we move around apps in Grafana and provides more functionality than a simple link.

Having said that, may be overkill for this simple link.

Reply:
I actually thought about it as a mix of both issues:
1. It was just a link
2. Since there isn't a lot of space, I thought adding a plugin extension would worsen the issue, and I assumed you wanted to check for a better design before opening it to every plugin

Sorry if I assumed too much 😅